plugins/clock/Clock.cc

```diff
@@ -20,35 +21,70 @@ using yarp::os::Network;
 namespace gzyarp
 {
 
-class Clock : public System, public ISystemPostUpdate
+class Clock : public System, public ISystemConfigure, public ISystemPostUpdate
 {
 public:
     Clock()
+        : m_portName("/clock")
+        , m_initialized(false)
+        , m_lastTimestampSent(0)
     {
-        m_portName = "/clock";
-        m_clockPort.open(m_portName);
+        std::cout << "===========> constructor" << std::endl;
     }
 
     ~Clock()
     {
+        std::cout << "===========> destructor" << std::endl;
```

Comment:
When debugging, always use `std::cerr`, `std::cout` may be buffered (even if in theory with `std::endl` you flush it).

Reply:
Ok thank you!